tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import yaml
+from plumbum import local
+from plumbum.cmd import git
+
+with open("copier.yml") as copier_fd:
+    COPIER_SETTINGS = yaml.safe_load(copier_fd)
+
+
+@pytest.fixture()
```

Comment:
Doing this once per session should be enough, right?

```suggestion
@pytest.fixture(scope="session")
```